CloudFormationLab.yml

```diff
@@ -47,20 +45,20 @@
       #     MapPublicIpOnLaunch: false
      
       
-      # InternetGateway:
-      #   Type: 'AWS::EC2::InternetGateway'
-      #   Properties:
-      #     Tags:
-      #       - Key: Name
-      #         Value: !Join [_, [!Ref 'AWS::StackName']]
-      #       - Key: Network
-      #         Value: Public
+      InternetGateway:
+        Type: 'AWS::EC2::InternetGateway' #creates gateway
+        Properties:
+          Tags:
```

Comment:
Recognize that tags are not required and are used to track resources

Reply:
Yeah! You would prefer I took them out?

Reply:
@tinaliu0-1629521 up to you, just pointing it out